highfidelity/hifi
# Patch for toggle 404 redirect (RC 73)

## CHANGES
- Patch for #14030 
    - The PR was incorrectly passed earlier.  This fixes the PR by persisting the user-configured change of disabling interstitial page by default
- Change for enabling/disabling interstitial mode
    - The test plan should now be for toggling interstitial mode will be using `Window.interstitialModeEnabled`.  Example: `Window.interstitialModeEnabled = true; // enables interstitialMode`

## TEST PLAN
- Launch Interface
- Enable developer menu in `Settings > Developer Menu`
- Open `Developer > Console` or `CTRL+ALT+J`
- In console, toggle on interstitial mode by using `Window.interstitialModeEnabled = true`
- Click `Edit > Reload Cache...` or `CTRL+R`
- Navigate to a domain using `GOTO` app
- While going to domain, interstitial page should pop up
- Navigate to a domain that would trigger a timeout, protocol mismatch, or not authorized error
- Interstitial page should pop up, then land you in the 404 redirect domain
## 
- In console, toggle off interstitial mode by using `Window.interstitialModeEnabled = false`
- Click `Edit > Reload Cache...` or `CTRL+R`
- Navigate to a domain using `GOTO` app
- While going to domain, interstitial page should not come up
- Navigate to a domain that would trigger a timeout, protocol mismatch, or not authorized error
- Interstitial page should not come up, then you should see a pop up dialog with the error you triggered
## 
- Try these a couple of times, also with closing/reopening the interface with differing settings from different sessions
